libs/utils/trace.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, platform, data_dir, events,
                  plots_prefix=''):
 
         # The platform used to run the experiments
-        self.platform = platform
+        self.platform = platform or {}
```

Comment:
I would say we can warn the use by checking that the first parameter, which will be `data_dir` is a string and not a dict... if it should be a dict we can show a DEPRECATED warning which notify the user about the new required interface and raise a ValueError exception asking for fixing the call.

Do you think this can work?

I think it's worth the change... in any case we wanna update that API to mandatory require only a trace file.